### PR TITLE
feat: Add support for Ordinal Measures

### LINKED ISSUE #733: Use Measures with Ordinal Scales
### User Story
As a user, I would like Visualize to properly parse measures with ordinal scale type defined in the Cube Creator.

### Definition of Done
Measures with ordinal scale type are treated differently than regular, numeric measures – e.g. it's not possible to use them as linear axis (for example as values in the bar chart).

### Additional info
This issue is about adding technical support for measures with ordinal scale types; tackling it will enable further advancements (such as #734).

Also see [dataset with ordinal measures](https://test.visualize.admin.ch/en/browse/dataset/https%3A%2F%2Fenvironment.ld.admin.ch%2Ffoen%2Ftest_ordinal_507%2F7?previous=%7B%22includeDrafts%22%3Atrue%2C%22order%22%3A%22SCORE%22%2C%22search%22%3A%22ord%22%7D&dataSource=Int).

### PR DESCRIPTION
**Pull request by bprusinowski**
Closes #733.

This PR adds support for measures with ordinal scale type. From what I understood from [Cube Schema docs](https://cube.link/#qudt-scaletype) and [Cube Designer docs](https://github.com/zazuko/cube-creator/wiki/3.-Cube-Designer#dimension-type) we need to have a distinction between ordinal dimensions and ordinal measures, mostly because ordinal dimensions are ordinal because of convenience (e.g. you want the Zurich canton to appear on top of other cantons, thus you add ordering) while ordinal measures are ordinal because of necessity (e.g. low, medium or high).

At this point I am not 100% sure if we **really** need to distinguish between them in Visualize – but there seems to be at least one use case where this will be required (we don't allow to color the map by using ordinal dimension, but we want to allow such coloring by ordinal measure in #734).

To make it easier I've also renamed current `Measure` to `NumericalMeasure`.

## How to test

- [x] Check `/en/browse/dataset/https%3A%2F%2Fenvironment.ld.admin.ch%2Ffoen%2Ftest_ordinal_507%2F7?previous=%7B%22includeDrafts%22%3Atrue%2C%22order%22%3A%22SCORE%22%2C%22search%22%3A%22ord%22%7D&dataSource=Int` and see that there are no NaNs
- [x] Create a chart from above dataset and check if it's only possible to create a table.

### CODE REVIEW

**Review by ptbrowne — APPROVED** [review]
LGTM 🎉 

I think that in the places where there are d => d.__typename === "NumericalDimension", we could replace them with a shared helper, that would also be a typeguard.

**Comment by ptbrowne** [comment]
Can you add an E2E test with the test dataset containing ordinals ?
I edited the description to add what I checked.
Only the table is possible to choose as a chart type, which seems OK.

**Comment by bprusinowski** [comment]
Thanks for adding "How to test" section, I forgot about this 🙇 I'll add E2E test 👍

File: cypress/integration/ordinal-measures.spec.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+beforeEach(() => {
+  cy.viewport("macbook-15");
+});
+
+describe("viewing a dataset with only ordinal measures", () => {
+  const key = "ePUgYyo622qS";
+  const config = testOrd507;
+
+  it("should retrieve dimension values properly", () => {
+    cy.visit(
```
Comment (ptbrowne) [inline cypress/integration/ordinal-measures.spec.ts:24]: No need to loadChartInLocalStorage here ?

**Review by ptbrowne — APPROVED** [review]
🎉

File: cypress/integration/ordinal-measures.spec.ts (hunk 1 above)
Comment (bprusinowski) [reply cypress/integration/ordinal-measures.spec.ts:24]: Nope, here we only look at dataset preview :)

**Pull request merged**